backend/core/src/listings/listings.controller.ts

```diff
@@ -60,14 +65,9 @@ export class ListingsController {
 
   @Get()
   @ApiOperation({ summary: "List listings", operationId: "list" })
-  @ApiImplicitQuery({
-    name: "jsonpath",
-    required: false,
-    type: String,
-  })
```

Comment:
So you're removing this capability entirely?

Reply:
I think this should have been removed/changed in https://github.com/CityOfDetroit/bloom/pull/18. I looked into whether Bloom adds this ApiImplicitQuery (or the new ApiQuery) decorator anywhere else, and it doesn't look like it.

Reply:
I see, so this wasn't Exygy that added it, but us?

Reply:
This `ApiImplicitQuery` decorator was there in the beginning (when `jsonpath` was the only variable in this method), when we forked Exygy's repo. Now that #18 made `jsonpath` just a field within `ListingsListQueryParams`, this decorator is incorrect